### DISCUSSION TITLE
Hackathon: 2024-07-23

### DISCUSSION THREAD

**Discussion by asteiker** [earthaccess-dev/earthaccess#772] [Hack days]
Reporting out on earthaccess hack days.

Please create a new comment (at the very bottom, use the text box with a button that says "comment", not "reply") to write about what you did today!

Please include links to GitHub issues or pull requests relevant to your work, and use those issues or pull requests as the "source of truth" for documenting the work; this makes future readers jobs easier :) It's OK to duplicate some information while writing a summary here!

Use the "reply" feature to have a discussion under any given comment.

**Comment by mfisher87** [comment]
Continued work to remove random collection sampling from our integration tests:

https://github.com/nsidc/earthaccess/pull/749

Not making as much progress as I'd like. As is usual for integration tests, there's a lot of wiring going on! Hoping to spend some time pair/group programming on this next hack day.

**Comment by chuckwondo** [comment]
@Sherwin-14 and I discussed his recent PR and how to address the failing unit tests, as summarized here: https://github.com/nsidc/earthaccess/pull/768#issuecomment-2246629415

**Comment by asteiker** [comment]
Coordinated on AGU submissions, per https://github.com/nsidc/earthaccess/discussions/775 and https://github.com/nsidc/earthaccess/discussions/774
